lib/install/management/rbac.go

```diff
@@ -110,7 +114,16 @@ func (am *AuthzManager) RoleList(ctx context.Context) (object.AuthorizationRoleL
 }
 
 func (am *AuthzManager) SetupRolesAndPermissions(ctx context.Context) ([]RBACResourcePermission, error) {
-	if _, err := am.CreateRoles(ctx); err != nil {
+	res, err := am.isPrincipalAnAdministrator(ctx)
+	if err != nil {
+		return nil, err
+	}
+	if res {
```

Comment:
As with the comment about silently dropping the effect of `--ops-grant-perms` I'd be inclined to make this an error.
Either the grant flag is not needed or this was not the user that they were intending.

Reply:
I don't think returning an error here is a good strategy. The check happens after the VCH has been created/re-configured. Since the user has administrative privileges the VCH will work correctly. I think the warning message is a better solution here.

Reply:
Ok. Hadn't remembered this was after the create.